yb_migrate/cmd/importData.go

```diff
@@ -63,6 +63,7 @@ var allTables []string
 var usePublicIp bool
 var targetEndpoints string
 var copyTableFromCommands = make(map[string]string)
+var LoadBalancerUsed bool // specifies whether load balancer is used in front of yb servers
```

Comment:
Start the variable name with a lower-case letter `loadBalancerUsed`.

Reply:
thanks, missed it.